appchallenge/menu.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Menu</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<link rel="stylesheet" href="details.css"/>
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<!-- Ted Pysh
+	index.html
+	info2439
+	Thoendel
+	9/8/19 
+	-->
+
+
+</head>
+<body>
+<div data-role="page">
+	<div data-role="header">
+	<h1>Menu</h1>
+	<img src="Sage-Bistro-Logo-2017-transparenent-small.png" id="logo">
+	</div>
+	<div data-role="header">
+	<a href="index.html" class="ui-btn-right"> Home</a>
+	<a href="survey.html" class="ui-btn"> Take Survey</a>
+	<a href="reservation.html" class="ui-btn"> Reservation</a>
+	<a href="maps.html" class="ui-btn"> Find Us</a>
+	</div>	
+<div role="main" class="ui-content">
+<h1> Lunch Menu </h1>
+<p> Below is listed our full lunch  menu.</p>
+	<h2>Entrees</h2>
```

Comment:
Good use of unordered lists here! You could go a step further with jquery mobile and look at using things like listview or inset to enhance the list. List dividers work well for headings.